### PR TITLE
Analysis Tools - Implemented 3 test cases JPSURV Application 

### PR DESCRIPTION
**Pull request by sgugulothu**
Analysis Tools - Implemented 3 test cases JPSURV Application

### CODE REVIEW

**Review by andreyeve — APPROVED** [review]

File: src/test/java/ANALYSIS_TOOLS/JPSurv/features/JPSurvHomePagePlaywright.feature [hunk 1]
```diff
@@ -156,4 +156,34 @@ Feature: These are the scenarios for the Home page on the JPSurv Application
     Then User clicks on the NHL checkbox
     Then User checks the relax proportionality and selects the cutpoint "5"
     Then User clicks on the submit button to calculate cohert and model specifications
-    Then Verify Cutpoint is coming next to Cohort and Model table headings
\ No newline at end of file
+    Then Verify Cutpoint is coming next to Cohort and Model table headings
+
+  @Smoke @satya @playwright @NCIATWP-6669
```
Comment (matakevin) [inline src/test/java/ANALYSIS_TOOLS/JPSurv/features/JPSurvHomePagePlaywright.feature:161]: If these new test cases are implemented then they should also have the regression tag

File: src/test/java/ANALYSIS_TOOLS/JPSurv/features/JPSurvHomePagePlaywright.feature [hunk 2]
```diff
@@ -156,4 +156,34 @@ Feature: These are the scenarios for the Home page on the JPSurv Application
...
+    Then User uploads file in the homepage
+    Then User selects start year of diagnosis
+    Then User selects end year of diagnosis
+    Then User selects interval of maximum years of diagnosis
+    Then User selects only CML checkbox only
+    Then User clicks on the submit button to calculate cohert and model specifications
+    Then User clicks on recalculate after selecting calendar year from "1980" to "1990"
+    Then Verify the user is getting result after recalculating
+
+  @Smoke @satya @playwright @NCIATWP-8065
```
Comment (matakevin) [inline src/test/java/ANALYSIS_TOOLS/JPSurv/features/JPSurvHomePagePlaywright.feature:173]: If these new test cases are implemented then they should also have the regression tag

File: src/test/java/ANALYSIS_TOOLS/JPSurv/features/JPSurvHomePagePlaywright.feature [hunk 3]
```diff
@@ -156,4 +156,34 @@ Feature: These are the scenarios for the Home page on the JPSurv Application
...
+  Scenario: Verify user is able to click on the footer links
+    Given User navigates to JPSurv home page
+    Then user clicks on the cancer control link and verify
+    Then user clicks on the accessibility link and verify it
+    Then user clicks on the FOIA link and verify it
+
+  @Smoke @satya @playwright @NCIATWP-6669
```
Comment (matakevin) [inline src/test/java/ANALYSIS_TOOLS/JPSurv/features/JPSurvHomePagePlaywright.feature:180]: If these new test cases are implemented then they should also have the regression tag

**Review by georgeb79 — APPROVED** [review]

**Review by matakevin — CHANGES_REQUESTED** [review]
Please add the tags and push changes

**Review by matakevin — APPROVED** [review]
LGTM

**Pull request merged**